app/config/permissions.js

```diff
@@ -65,6 +65,7 @@ const groups = [
       'view-document-version-info',
       'view-documents-before-release',
       'view-decisions-before-release',
+      'view-confidential-cases',
```

Comment:
This is what was requested in the ticket, but is somewhat confusing.
A `case` or `decisionmaking-flow` itself is never confidential, subcases are.
Is a case with 5 subcases where 1 subcase is confidential a confidential case?
According to current logic, it is.
In that light the permission name does make sense.

I suggest:
- in any case, add the permission to the commented list above
- keep the name but mention how a case can be considered confidential when 1 of its subcase is confidential.